test/associations_test.rb

```diff
@@ -44,7 +44,25 @@ def test_multiple_associations
       }
 
       PersonSerializer.new(peggy, includes: [:colleagues, :superior])
-        .associations.must_equal expected
+        .associations
+        .must_equal expected
+    end
+
+    def test_default_associations
+      peggy = Person.new
+      roger = Person.new(id: 2, name: 'Roger', occupation: 'Chief')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.